### PR TITLE
[Components with more than 1 service] LPS-184060 Refactor Editor components from frontend-editor module part II

### PR DESCRIPTION
**Pull request by MrJohn1911**

### CODE REVIEW

**Comment by MrJohn1911** [comment]
ci:test:startup

**Comment by MrJohn1911** [comment]
ci:test:frontend

**Comment by Kvale1733** [comment]
@MrJohn1911 No regressions found.

File: modules/apps/frontend-editor/frontend-editor-alloyeditor-web/src/main/java/com/liferay/frontend/editor/alloyeditor/web/internal/AlloyEditorBBCodeEditor.java [hunk 1]
```diff
@@ -27,47 +24,14 @@
...
-		return PortalWebResourceConstants.RESOURCE_TYPE_EDITOR_ALLOYEDITOR;
-	}
+@Component(property = "name=alloyeditor_bbcode", service = Editor.class)
+public class AlloyEditorBBCodeEditor extends BaseAlloyEditor {
 
 	@Activate
 	protected void activate(Map<String, Object> properties) {
-		_name = (String)properties.get("name");
-	}
+		String name = (String)properties.get("name");
```
Comment (rafaprax) [inline modules/apps/frontend-editor/frontend-editor-alloyeditor-web/src/main/java/com/liferay/frontend/editor/alloyeditor/web/internal/AlloyEditorBBCodeEditor.java:32]: Hey @MrJohn1911, please inline this statement. Thx

**Review by rafaprax — CHANGES_REQUESTED** [review]

**Comment by MrJohn1911** [comment]
ci:test:sf

**Comment by MrJohn1911** [comment]
ci:test:relevant

**Comment by MrJohn1911** [comment]
ci:test:startup

**Comment by MrJohn1911** [comment]
ci:test:frontend

**Comment by zhangyan0701** [comment]
ci:test:startup

**Comment by zhangyan0701** [comment]
ci:test:frontend

**Comment by catarineaguiar** [comment]
no regressions found.

**Pull request closed without merging**